packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -54,22 +60,29 @@ export class DeltaStorageService implements IDeltaStorageService {
         to?: number): Promise<ISequencedDocumentMessage[]> {
         const query = querystring.stringify({ from, to });
 
-        let headers: { Authorization: string } | null = null;
+        const headers: OutgoingHttpHeaders = {
+            "x-correlation-id": uuid.v4(),
+        };
 
         const storageToken = await this.tokenProvider.fetchStorageToken(
             tenantId,
             id,
         );
 
         if (storageToken) {
-            headers = {
-                Authorization: `Basic ${fromUtf8ToBase64(`${tenantId}:${storageToken.jwt}`)}`,
-            };
+            headers.Authorization = `Basic ${fromUtf8ToBase64(`${tenantId}:${storageToken.jwt}`)}`;
```

Comment:
Nit: Why di we need this change?

Reply:
The initial value of headers was null. Now the initial value of headers is { "x-correlation-id": uuid.v4() } (on line 63). So it needs to add a key to dictionary instead of giving a value to it.